emcc.py

```diff
@@ -462,7 +462,7 @@ def is_supported(f):
         # lld allows various flags to have either a single -foo or double --foo
         if f.startswith(flag) or f.startswith('-' + flag):
           diagnostics.warning('linkflags', 'ignoring unsupported linker flag: `%s`', f)
-          return False, takes_arg
+          return False, takes_arg and '=' not in f
```

Comment:
I don't follow this. Perhaps a comment about why `=` is special?

Reply:
Done